src/components/global/Hero/Hero.jsx

```diff
@@ -0,0 +1,47 @@
+// Material UI Imports
+import Box from '@mui/material/Box';
+import Button from '@mui/material/Button';
+import Typography from '@mui/material/Typography';
+//import component specific styles
+import { heroStyles } from './styles';
+
+const Hero = (props) => {
```

Comment:
Personally, I would prefer destructuring these `props` to `{ pageName, heroImage, heroText }` that way any dev knows exactly what this component expects - without having to dig into it.  

Unless this way is easier for you to tell what is imported through the styles file vs passed down as a prop, then leave as is - just an opinion.

---

Either way, eslint will continue to whine about it not knowing what type these props are.  Do you happen to be familiar with PropTypes? Declaring the types using PropTypes would solve, and give other devs another clue to not only what props this component expects - but what those props should be.

Reply:
Wasn't familiar with PropTypes but I imported it (already a dependency) and added propTypes to the bottom of Hero right before the export. Please let me know if it looks correct. It seems to work 🤷‍♂️